app/appContext.js

```diff
@@ -1,4 +1,6 @@
 const resourcesData = require('./data/resources');
 const resourceModel = require('./models/resourcesModel')({ data: resourcesData });
+const categoriesData = require('./data/categories');
+const categoryModel = require('./models/categories-model')({ data: categoriesData });
```

Comment:
For the categories not sure this needs the full model findAll treatment ... maybe just pass the categories data to the page to be rendered.